### DISCUSSION TITLE
What is the default value for `inputFormat` ?

### DISCUSSION THREAD

**Discussion by nichita-pasecinic** [mui/mui-x#6492] [Archive]
What is the default value for `inputFormat` ?

does the adapter use the locale ?

**Comment by LukasTy** [comment]
We do not control its value explicitly.
By default it is taken from `keyboardDate` (for `DatePicker` component) field from the adapter.
dayjs [example](https://github.com/dmtrKovalenko/date-io/blob/master/packages/dayjs/src/dayjs-utils.ts#L52)

**Reply by nichita-pasecinic** [reply]
What `L` stands for at `dayjs` ? 
I assume is localized by default in dayjs ?

**Reply by LukasTy** [reply]
> What `L` stands for at `dayjs` ?

Yes, it is [localized date format](https://day.js.org/docs/en/display/format#:~:text=02%3A18%20PM-,L,-MM/DD/YYYY)—it depends on the selected locale.
